.github/workflows/production-deploy.yml

```diff
@@ -7,56 +7,39 @@ on:
         required: true
 
-env:
-  DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
-  ERICA_HOST: ${{ secrets.ERICA_HOST_PRODUCTION }}
-
 jobs:
-  deploy-production:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Login to container repository
-        uses: docker/login-action@v1
-        with:
-          registry: ${{ env.DOCKER_REGISTRY }}
-          username: ${{ secrets.REGISTRY_USER }}
-          password: ${{ secrets.REGISTRY_PWD }}
-
-      - name: Pull deploy images
-        run: docker pull ${{ env.DOCKER_REGISTRY }}/steuerlotse_deployment
-        shell: bash
-
-      - name: Run deployment
-        env:
-          NAMESPACE: erica-production
-          IMAGE_TAG: ${{ github.event.inputs.imageTag }}
-        run: |
-          docker run \
-          -e DOCKER_REGISTRY='${{ env.DOCKER_REGISTRY }}' \
-          -e ERICA_SSH_KEY='${{ secrets.CI_ERICA_SSH_KEY }}' \
-          -e ERICA_HOST='${{ env.ERICA_HOST }}' \
-          -e REGISTRY_PWD='${{ secrets.REGISTRY_PWD }}' \
-          -e CERT_PIN='${{ secrets.CERT_PIN_PRODUCTION }}' \
-          -e QUEUE_URL='${{ secrets.QUEUE_URL_PRODUCTION }}' \
-          -e ERICA_DATABASE_URL='${{ secrets.ERICA_DATABASE_URL_PRODUCTION }}' \
-          -e KUBECONFIG_BASE64='${{ secrets.CI_KUBECONFIG_B64 }}' \
-          -e NAMESPACE='${{ env.NAMESPACE }}' \
-          -e IMAGE_TAG='${{ env.IMAGE_TAG }}' \
-          ${{ env.DOCKER_REGISTRY }}/steuerlotse_deployment erica
-        shell: bash
-
-      - name: Logout of docker container
-        run: docker logout ${{ env.DOCKER_REGISTRY }}
-
-      - name: Report Deployment
-        uses: satak/webrequest-action@v1.2.3
-        env:
-          IMAGE_TAG: ${{ github.event.inputs.imageTag }}
-        with:
-          url: ${{ secrets.METRICS_DEPLOYMENT_WEBHOOK_URL }}
-          method: POST
-          payload: '{"project": "Erica", "version_identifier": "${{ env.IMAGE_TAG }}", "environment": "production", "link": "https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"}'
-          headers: '{"Authorization": "Token ${{ secrets.METRICS_WEBHOOK_TOKEN }}"}'
+  syseleven:
+    uses: ./.github/workflows/reusable-deploy-syseleven.yml
+    with:
+      imageTag: ${{ github.event.inputs.imageTag }}
+      environment: production
+    secrets:
+      DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+      REGISTRY_USER: ${{ secrets.REGISTRY_USER }}
+      REGISTRY_PWD: ${{ secrets.REGISTRY_PWD }}
+      ERICA_SSH_KEY: ${{ secrets.CI_ERICA_SSH_KEY }}
+      ERICA_HOST: ${{ secrets.ERICA_HOST_PRODUCTION }}
+      CERT_PIN: ${{ secrets.CERT_PIN_PRODUCTION }}
+      QUEUE_URL: ${{ secrets.QUEUE_URL_PRODUCTION }}
+      ERICA_DATABASE_URL: ${{ secrets.ERICA_DATABASE_URL_PRODUCTION }}
+      KUBECONFIG_B64: ${{ secrets.CI_KUBECONFIG_B64 }}
+      METRICS_DEPLOYMENT_WEBHOOK_URL: ${{ secrets.METRICS_DEPLOYMENT_WEBHOOK_URL }}
+      METRICS_WEBHOOK_TOKEN: ${{ secrets.METRICS_WEBHOOK_TOKEN }}
```

Comment:
You specify these here but I do not see them in reusable-deploy-syseleven.yml? 

I think we should a) add the report deployment step back into this workflow or b) add it to the separated actions. I would prefer a because this would only report once. wdyt?

Reply:
Ah yes, thanks, that's an oversight. At one point I had it inside the syseleven workflow, but then realised we'd only want to report a deployment once both succeeded.